capstone-ui/build/index.html

```diff
@@ -1 +1 @@
-<!doctype html><html lang="en"><head><meta charset="UTF-8"/><link rel="icon" type="image/svg+xml" href="/src/favicon.svg"/><meta name="viewport" content="width=device-width,initial-scale=1"/><title>M&M</title><script defer="defer" src="/static/js/main.69800221.js"></script><link href="/static/css/main.2215139c.css" rel="stylesheet"></head><body><div id="root"></div><script type="module" src="./src/index.js"></script></body></html>
\ No newline at end of file
+<!doctype html><html lang="en"><head><meta charset="UTF-8"/><meta name="viewport" content="width=device-width,initial-scale=1"/><title>M&M</title><script defer="defer" src="/static/js/main.69800221.js"></script><link href="/static/css/main.2215139c.css" rel="stylesheet"></head><body><div id="root"></div><script type="module" src="./src/index.js"></script></body></html>
```

Comment:
Nit: 
Does this file changes when you "build" the front end? 
If yes, then it should be on the git ignore.